recipes/simdjson/all/CMakeLists.txt

```diff
@@ -1,8 +0,0 @@
-cmake_minimum_required(VERSION 3.9)
```

Comment:
Need to be replicated to 0.4.6. conan_basic_setup is required

Reply:
Can you elaborate? There is a file named recipes/simdjson/0.4.6/test_package/CMakeLists.txt that includes conan_basic_setup.

Reply:
Oh. I figured it out.

Reply:
Fixed.